Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -157,6 +163,18 @@ - (BOOL)rowAtIndexPathIsOnlyRowVisibleInSection:(NSIndexPath*)indexPath {
     }
 }
 
+/**
+ *  Check whether or not a section is going in or out of view.
+ *
+ *  @param indexPath The index path of the row which will or did end displaying.
+ *
+ *  @return @c YES if that section isn't displaying or the given row is the only one visible in its section, otherwise @c NO.
+ */
+- (BOOL)isVisibilityTransitioningForRowIndexPath:(NSIndexPath*)indexPath {
+    return ![self isDisplayingCellsForSection:indexPath.section]
```

Comment:
prior uses notwithstanding, I think it is true in both contexts, which I tried to explain in the above conditions.  let me try to restate more explicitly:

- section isn't listed in `indexPathsForVisibleCells`
  - `willDisplayCell`: we're about to display it
  - `didEndDisplayingCell`: the section scrolled out of view
- `indexPath.row` is the only one visible in its section
  - `willDisplayCell`: this will be the first row to appear in its section
  - `didEndDisplayingCell`: this was the last cell to scroll away in its section

having said all that, since this method is only used one place i can give it a name more specific to the context in which its currently used.